llvm/test/CodeGen/X86/avx512-intrinsics-fast-isel.ll

```diff
@@ -21,7 +21,7 @@ define zeroext i16 @test_mm512_kunpackb(<8 x i64> %__A, <8 x i64> %__B, <8 x i64
 ; X86-NEXT:    kunpckbw %k0, %k1, %k1
 ; X86-NEXT:    vpcmpneqd 72(%ebp), %zmm3, %k0 {%k1}
 ; X86-NEXT:    kmovw %k0, %eax
-; X86-NEXT:    movzwl %ax, %eax
+; X86-NEXT:    # kill: def $ax killed $ax killed $eax
```

Comment:
I don't see problem from test case, because the high bits are zeros anyway.
I'm not sure if it's correct in general. The `zeroext` requires the callee to zero-extend the return type. I think we still need an explicit zero-extend for `i8/16` type if the high bits are not zero.

See LangRef about zeroext:
> This indicates to the code generator that the parameter or return value should be zero-extended to the extent required by the target’s ABI by the caller (for a parameter) or the callee (for a return value).